src/services/azure/constants.rs

```diff
@@ -0,0 +1,15 @@
+// Env values used in azure services.
+//AZURE_STORAGE_ACCOUNT = Credential.access_name
+#[allow(dead_code)]
+pub const AZURE_STORAGE_ACCOUNT: &str = "AZURE_STORAGE_ACCOUNT";
+//AZURE_STORAGE_ACCOUNT = Credential.shared_key
+#[allow(dead_code)]
+pub const AZURE_STORAGE_KEY: &str = "AZURE_STORAGE_KEY";
+#[allow(dead_code)]
+pub const CONTENT_MD5: &str = "Content-MD5";
+#[allow(dead_code)]
+pub const MS_DATE: &str = "x-ms-date";
```

Comment:
How about scope all ms related headers into a header mod?

So that we can use `header::MS_DATE`, `header::MS_VERSION`?

Reply:
Ok